controllers/articles.controller.js

```diff
@@ -1,8 +1,9 @@
-const model = require("../models");
+const model = require("../models").articles;
```

Comment:
Ah, I see what's happened here. It would be better to do something like const { getArticles } and then the path to wherever that model is. At the moment, it's hard to tell the purpose of model, and the path of ("../models").articles is hard to comprehend.